hedera-node/src/test/java/com/hedera/services/store/models/TokenConversionTest.java

```diff
@@ -1,157 +1,190 @@
 import com.hedera.services.state.submerkle.EntityId;
 import com.hedera.services.state.submerkle.RichInstant;
-import com.hedera.services.store.AccountStore;
 import com.hedera.test.factories.scenarios.TxnHandlingScenario;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
 
-import java.util.List;
+import java.time.Instant;
 
+import static com.hedera.services.store.models.TokenConversion.fromMerkle;
+import static com.hedera.services.store.models.TokenConversion.fromMerkleUnique;
+import static com.hedera.services.store.models.TokenConversion.fromToken;
+import static com.hedera.services.store.models.TokenConversion.fromUniqueToken;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.mockito.BDDMockito.given;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
 
-@ExtendWith(MockitoExtension.class)
 class TokenConversionTest {
+	private static final long tokenNum = 777L;
+	private static final long serialNum = 10L;
+	private static final Id tokenId = new Id(0, 0, tokenNum);
+	private static final Token token = new Token(tokenId);
+	private static final EntityId owner = new EntityId(0, 0, 3);
+	private static final byte[] metadata = "Test NFT".getBytes();
+	private static final UniqueToken uniqueToken = new UniqueToken(tokenId, serialNum);
+	private static final RichInstant timestamp = RichInstant.fromJava(Instant.ofEpochSecond(1_234_567L));
+	private static final long expiry = 1_234_567L;
+	private static final long treasuryAccountNum = 2_234L;
+	private static final long autoRenewAccountNum = 3_234L;
+	private static final Id treasuryId = new Id(0, 0, treasuryAccountNum);
+	private static final Id autoRenewId = new Id(0, 0, autoRenewAccountNum);
+	private static final Account treasuryAccount = new Account(treasuryId);
+	private static final Account autoRenewAccount = new Account(autoRenewId);
+
+	private static final JKey kycKey = TxnHandlingScenario.TOKEN_KYC_KT.asJKeyUnchecked();
+	private static final JKey freezeKey = TxnHandlingScenario.TOKEN_FREEZE_KT.asJKeyUnchecked();
+	private static final JKey supplyKey = TxnHandlingScenario.TOKEN_SUPPLY_KT.asJKeyUnchecked();
+	private static final JKey wipeKey = TxnHandlingScenario.TOKEN_WIPE_KT.asJKeyUnchecked();
+	private static final JKey adminKey = TxnHandlingScenario.TOKEN_ADMIN_KT.asJKeyUnchecked();
+	private static final JKey feeScheduleKey = TxnHandlingScenario.TOKEN_ADMIN_KT.asJKeyUnchecked();
+	private static final long tokenSupply = 777L;
+	private static final long autoRenewPeriod = 1234L;
+	private static final String name = "Testing123";
+	private static final String symbol = "T123";
+	private static final String memo = "memo";
+	private static final boolean freezeDefault = true;
+	private static final boolean kycGrantedDefault = true;
```

Comment:
nit: these vars are all constants, maybe use the constant convention to name them.

Reply:
understood. Will update them in the main PR for branch `ci-enable-2143` - PR #2221